pep-8104.rst

```diff
@@ -0,0 +1,162 @@
+PEP: 8104
+Title: 2023 Term steering council election
```

Comment:
```suggestion
Title: 2023 Term Steering Council election
```

SC is a proper noun, so it certainly doesn't make sense to not capitalize it if "2023 Term" is considered one and capitalized accordingly